src/floodfill.rs

```diff
@@ -3,34 +3,19 @@ use std::collections::{BinaryHeap, HashSet};
 use crate::priority_queue::PriorityQueueItem;
 use crate::shared::{Cost, EdgePT, EdgeWalk, NodeID};
 use smallvec::SmallVec;
-use std::sync::Arc;
 
 pub fn floodfill(
-    (
-        graph_walk,
-        start,
-        node_values_1d,
-        travel_time_relationships,
-        subpurpose_purpose_lookup,
-        graph_pt,
-        trip_start_seconds,
-        init_travel_time,
-        count_original_nodes,
-        node_values_padding_row_count,
-        target_destinations_vector,
-    ): (
-        &Vec<SmallVec<[EdgeWalk; 4]>>,
-        NodeID,
-        &Vec<i32>,
-        &Arc<Vec<i32>>,
-        &[i8; 32],
-        &Vec<SmallVec<[EdgePT; 4]>>,
-        i32,
-        Cost,
-        u32,
-        u32,
-        &Vec<u32>,
-    ),
+    graph_walk: &Vec<SmallVec<[EdgeWalk; 4]>>,
```

Comment:
It could be nice to take a `Request` struct and/or organize these based on which vary in the parallel loop